.github/workflows/ci.yml

```diff
@@ -130,6 +130,7 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           files: coverage.xml
+          flags: ${{ matrix.browser }}-${{ matrix.selenium }}
```

Comment:
Quick question before merging, could it make sense to split this and have 2 flags?
```yaml
flags: ${{ matrix.browser }},selenium-${{ matrix.selenium }}
```
=> `firefox` and `selenium-3`

Reply:
According to the https://docs.codecov.com/docs/flags you can only specify 1 flag in a filter in the UI (I'm maybe mistaken).

Filtering by specifying flags in URL could also be done using `https://app.codecov.io/gh/your-org/your-repo?flags=frontend,backend` URL.

Then we need to specify 3 tags:

* browser+selenium_version
* browser
* selenium_version

Do you agree?